src/app.ts

```diff
@@ -113,6 +114,40 @@ export async function getApp(configOverrides: ConfigOverrides = {}) {
     },
   })
 
+  await app.register(fastifyCors, {
+    origin: (origin, cb) => {
+      if (origin === undefined) {
+        cb(null, false)
+        return
+      }
+
+      const hostname = new URL(origin).hostname
+
+			// CE pass-through
+			if (hostname.includes('cteng')) {
```

Comment:
isn't this easy to fake?

Reply:
Not sure, but feel free to share any suggestion

Reply:
I mean, you can create `fakeserver-cteng-totally-not-a-scam.com` host, and it would pass this check.
Don't we know exact hostname for production ditto environment?

Reply:
Yeah, it's `https://lok-cteng-main.live.lokalise.cloud`. We could test it using a Regex for `https://lok-cteng-*.live.lokalise.cloud` WDYT?

Reply:
regexes are expensive, I would rather not do that check on every call.
Probably we can assume that if it ends with `.lokalise.cloud`, then it's safe.

Reply:
Deal! :+1: